9.4/root/usr/bin/container-entrypoint

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
```

Comment:
@mfojtik I appreciate your focus on detail, but I don't think keeping the shebang format same is something we need to keep same across images, in case it works the same.